<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchOperateDialog</class>
 <widget class="QDialog" name="BatchOperateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QLabel" name="_patternLabel">
         <property name="text">
          <string>键值模式:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="_patternCountLineEdit"/>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="_patternSeparatorLineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="_batchComboBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="_batchOplabel">
         <property name="text">
          <string>批量操作:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="_patternCountLabel">
         <property name="text">
          <string>模式计数:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="_patternSeparatorLabel">
         <property name="text">
          <string>模式分割:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="QLineEdit" name="_patternlineEdit"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <spacer name="_verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QProgressBar" name="_progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTextBrowser" name="_textBrowser"/>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="_buttonHorizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>238</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="_cancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_okPushButton">
         <property name="text">
          <string>Execute</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_exitPushButton">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
